### PR TITLE
add webview support

### PR DESCRIPTION
**Pull request by aecast**
Allow notifications to open a webview activity from the app instead of switching to a browser.

### CODE REVIEW

File: src/main/java/com/mixpanel/android/mpmetrics/MixpanelWebViewActivity.java [hunk 1]
```diff
@@ -1,6 +1,36 @@
...
 import android.app.Activity;
+import android.os.Bundle;
+import android.webkit.WebChromeClient;
+import android.webkit.WebSettings;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+
+import com.mixpanel.android.R;
 
 public class MixpanelWebViewActivity extends Activity {
```
Comment (jbwyme) [inline src/main/java/com/mixpanel/android/mpmetrics/MixpanelWebViewActivity.java]: why did this class already exist? what was it used for?

Comment (aecast) [reply src/main/java/com/mixpanel/android/mpmetrics/MixpanelWebViewActivity.java]: It didn't, I think the class creation is either in another PR or wasn't tracked

Comment (jbwyme) [reply src/main/java/com/mixpanel/android/mpmetrics/MixpanelWebViewActivity.java]: ah, it should be in this one though shouldn't it? along with whatever code in the router that calls it?

Comment (aecast) [reply src/main/java/com/mixpanel/android/mpmetrics/MixpanelWebViewActivity.java]: yeah found it on other PR going to fix up

File: src/main/java/com/mixpanel/android/mpmetrics/MixpanelPushNotification.java [hunk 2]
```diff
@@ -31,6 +31,7 @@
 
 public class MixpanelPushNotification {
     protected final String LOGTAG = "MixpanelAPI.MixpanelPushNotification";
+    protected final int ROUTING_REQUEST_CODE = 1;
```
Comment (jbwyme) [inline src/main/java/com/mixpanel/android/mpmetrics/MixpanelPushNotification.java]: add comment about what this does and why we chose 1

File: src/main/java/com/mixpanel/android/mpmetrics/MixpanelPushNotification.java [hunk 3]
```diff
@@ -188,8 +189,8 @@ protected void buildNotificationFromData() {
         final PendingIntent contentIntent = PendingIntent.getActivity(
                 context,
                 0,
-                data.intent,
-                PendingIntent.FLAG_UPDATE_CURRENT
+                getRoutingIntent(data.onTap, "notificationClick", "notification"),
```
Comment (jbwyme) [inline src/main/java/com/mixpanel/android/mpmetrics/MixpanelPushNotification.java]: as mentioned in another PR, I think we should just be passing the `target` (as an enum) to `getRoutingIntent` and when the target is `notification`, we don't need an id or label.

**Pull request merged**